contrib/platform/src/com/sun/jna/platform/win32/Variant.java

```diff
@@ -789,6 +790,7 @@ public VariantArg() {
          */
         public VariantArg(Pointer pointer) {
             super(pointer);
+            variantArg = new VARIANT[1];
```

Comment:
I am not sure about this, as the super call is now done after the field is initialized.